wpimath/src/main/java/edu/wpi/first/math/filter/SlewRateLimiter.java

```diff
@@ -14,53 +14,107 @@
  * edu.wpi.first.math.trajectory.TrapezoidProfile} instead.
  */
 public class SlewRateLimiter {
-  private final double m_rateLimit;
-  private double m_prevVal;
-  private double m_prevTime;
+    private final double m_rateLimit;
```

Comment:
The singular rate limit should be removed. The case of having a symmetric limit is just the same as having forward and backward be equal.